src/screens/Home/index.tsx

```diff
@@ -14,43 +14,27 @@ import {
   LogBox,
 } from 'react-native';
 import { SvgXml } from 'react-native-svg';
-import { postIconOutlined, searchIcon } from '../../svg/svg-xml-list';
+import { postIconOutlined } from '../../svg/svg-xml-list';
 import FloatingButton from '../../components/FloatingButton';
 import useAuth from '../../hooks/useAuth';
 import Explore from '../Explore';
 import GlobalFeed from '../GlobalFeed';
 import { getStyles } from './styles';
 import CreatePostModal from '../../components/CreatePostModal';
 import CustomTab from '../../components/CustomTab';
-import { useNavigation } from '@react-navigation/native';
-import type { NativeStackNavigationProp } from '@react-navigation/native-stack';
 import { useTheme } from 'react-native-paper';
 import type { MyMD3Theme } from '../../providers/amity-ui-kit-provider';
 LogBox.ignoreAllLogs(true);
 export default function Home() {
   // const { t, i18n } = useTranslation();
   const styles = getStyles();
   const { client } = useAuth();
-  const theme = useTheme() as MyMD3Theme ;
-  
+  const theme = useTheme() as MyMD3Theme;
+
   const [activeTab, setActiveTab] = useState(1);
   const [isVisible, setIsVisible] = useState(false);
 
   const [createPostModalVisible, setCreatePostModalVisible] = useState(false);
-  const navigation = useNavigation<NativeStackNavigationProp<any>>();
```

Comment:
Yes. We don't use that anymore. 
Before that, we are adding navigation Header ( e.g. Header title, Back Button, Save Button, Menu Button, etc. ) after rendering the Home screen. 

It causes error (5th Image I mentioned ) . 
So I moved header to Navigator. So that Header will render before the screen.